### DISCUSSION TITLE
Customization of the default renovate.json file

### DISCUSSION THREAD

**Discussion by oli271078** [renovatebot/renovate#26837] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

renovateVersion: 37.52.0

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### What is your question?

The first time renovate is executed on a repository, it creates a merge request with the renovate.json file. This file content is:
```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
}
```

Is it possible to configure renovate so that the file created in each repository already contains some specific configuration ?
For example, i would like that the renovate.json file created in the new repositories will be
```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "labels": [
    "update-dependencies"
  ]
}
```

Thanks for your response

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by secustor** [comment, marked as answer]
See [`onboardingConfig`](https://docs.renovatebot.com/self-hosted-configuration/#onboardingconfig).

**Reply by oli271078** [reply]
Perfect ! Thanks !!
